internal/connector/pingfederate/resources/pingfederate_extended_properties.go

```diff
@@ -0,0 +1,52 @@
+package resources
+
+import (
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/common"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingFederateExtendedPropertiesResource{}
+)
+
+type PingFederateExtendedPropertiesResource struct {
+	clientInfo *connector.PingFederateClientInfo
+}
+
+// Utility method for creating a PingFederateExtendedPropertiesResource
+func ExtendedProperties(clientInfo *connector.PingFederateClientInfo) *PingFederateExtendedPropertiesResource {
+	return &PingFederateExtendedPropertiesResource{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingFederateExtendedPropertiesResource) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+
+	importBlocks := []connector.ImportBlock{}
+
+	l.Debug().Msgf("Generating Import Blocks for all %s resources...", r.ResourceType())
+
+	extendedPropertiesId := "extended_properties_singleton_id"
+	extendedPropertiesName := "Extended Properties"
+
+	commentData := map[string]string{
+		"Resource Type": r.ResourceType(),
+		"Singleton ID":  "This resource is a singleton, so the value of 'ID' in the import block does not matter - it is just a placeholder and required by terraform.",
```

Comment:
Would probably be useful to make this string a constant somewhere

Reply:
Will do.